packages/salesforcedx-vscode-lightning/src/index.ts

```diff
@@ -6,7 +6,7 @@
  */
 
 import { shared as lspCommon } from '@salesforce/lightning-lsp-common';
-import { TelemetryService } from '@salesforce/salesforcedx-utils-vscode/out/src/telemetry';
+import { TelemetryService } from '@salesforce/salesforcedx-utils-vscode/out/src';
```

Comment:
I think we'll need to update the soql extension too https://github.com/forcedotcom/salesforcedx-vscode/blob/8c1348493cb62410ff5a39d132b5f00eea564a7b/packages/salesforcedx-vscode-soql/src/telemetry/index.ts#L7